docs/_sass/_docsearch.scss

```diff
@@ -0,0 +1,45 @@
+.searchbox {
+  display: inline-block;
+  font-size: 1rem;
+
+  .searchbox__input {
+    padding: 5px 5px 5px 29px;
+    border: none;
+    border-radius: 5px;
+    color: #444;
+
+    &::-webkit-input-placeholder {
+      color: #aaa;
+    }
+
+    &:-ms-input-placeholder {
+      color: #aaa;
+    }
+
+    &::placeholder {
+      color: #aaa;
+    }
+  }
+}
+
+.algolia-autocomplete .ds-dropdown-menu [class^="ds-dataset-"] {
+}
+
+.algolia-autocomplete .algolia-docsearch-suggestion--category-header {
+  background-color: #444;
+  color: #ddd;
+  padding: 0.35em;
+}
+
+.algolia-autocomplete .algolia-docsearch-suggestion--subcategory-column {
+  color: #444;
+}
+
+.algolia-autocomplete .algolia-docsearch-suggestion--highlight {
+  background-color: #fc0;
+  color: #222;
+}
+
+.algolia-autocomplete .algolia-docsearch-suggestion--text .algolia-docsearch-suggestion--highlight {
+  box-shadow: inset 0 -2px 0 0 #fc0;
+}
```

Comment:
if this is not an auto-generated page, can we have these nested?
```sass
.algolia-autocomplete {
  .algolia-docsearch-suggestion--category-header { }
  .algolia-docsearch-suggestion--subcategory-column { }
}
```

Reply:
fixed